docs/index.rst

```diff
@@ -1,6 +1,54 @@
-=======
-Home
-=======
+.. meta::
+  :description: Half documentation 
+  :keywords: Half, APIs, ROCm, documentation
+
+*************************
+Half documentation
+*************************
+
+Half-Precision Floating Point library (Version 1.12.0)
+------------------------------------------------------
+
+Half is a C++ header-only library to provide an IEEE 754 conformant 16-bit half-precision floating point type along with corresponding arithmetic operators, type conversions and common mathematical functions. It aims for both efficiency and ease of use, trying to accurately mimic the behaviour of the builtin floating point types at the best performance possible.
```

Comment:
```suggestion
Half is a C++ header-only library that provides an IEEE 754 conformant 16-bit half-precision floating point type along with corresponding arithmetic operators, type conversions, and common mathematical functions. It aims for both efficiency and ease of use, trying to accurately mimic the behavior of the built-in floating point types at the best performance possible.
```

Reply:
And shouldn't there be a link to a github repository after the intro?

Reply:
> And shouldn't there be a link to a github repository after the intro?

Thanks so much, Leo. I think the decision is to not publish Half API documentation on the portal. @yhuiYH Could you please confirm?